tasks/apm-inject-install.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Include APM host injection Debian install tasks
+  include_tasks: pkg-debian/install-apm-inject-latest.yml
+  when: not ansible_check_mode and ansible_facts.os_family == "Debian" and datadog_apm_instrumentation_enabled != ""
+
+- name: Include APM host injection RedHat install tasks
+  include_tasks: pkg-redhat/install-apm-inject-latest.yml
+  when: not ansible_check_mode and ansible_facts.os_family in ["RedHat", "Rocky", "AlmaLinux"] and datadog_apm_instrumentation_enabled != ""
+
+#- name: Check if dd-host-install needs to run
+#  command: dd-host-install --dry-run
+#  register: datadog_apm_instrumentation_rc
+#  ignore_errors: true
+
+- name: Run APM host injection setup script
+  command: dd-host-install --no-config-change --no-agent-restart
```

Comment:
Should this have `--no-agent-config-change` instead of `--no-config-change`?

Reply:
the flag for this `dd-host-install` is called `--no-config-change`